environment.yml

```diff
@@ -18,7 +19,6 @@ dependencies:
 - jupyter
 - pytest
 - pytest-cov
-- sphinx
 - joblib
 - psutil
 - numpydoc
```

Comment:
why keep numpydoc? you decided to remove all packages related to doc building?

Reply:
we use it in `test_docstring_parameters.py`